modules/ossm-adding-project-using-smm-resource-cli.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-create-mesh.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-adding-project-using-smm-resource-cli_{context}"]
+= Adding a project to the service mesh using the ServiceMeshMember resource with the CLI
+
+.Prerequisites
+* An installed, verified {SMProductName} Operator.
+* Name of the `ServiceMeshControlPlane` resource and the name of the project it belongs to.
+* Name of the project you want to add to the mesh.
+* A service mesh administrator must explicitly grant access to the service mesh. Administrators can grant users permissions to access the mesh by assigning them the `mesh-user` user role.
+
+.Procedure
+
+. Log in to the {product-title} CLI.
+
+. Create the YAML file for the `ServiceMeshMember` manifest. The manifest adds the `my-application` project to the service mesh that was created by the `ServiceMeshControlPlane` resource deployed in the `istio-system` namespace:
++
+[source,yaml]
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshMember
+metadata:
+  name: default
+  namespace: my-application
+spec:
+  controlPlaneRef:
+    namespace: istio-system
+    name: basic
+----
+
+. Apply the YAML file to create the `ServiceMeshMember` resource:
++
+[source,terminal]
+----
+oc apply -f <file-name>
+----
+
+. After the `ServiceMeshMember` resource is created, verify the project namespace is part of the mesh. Confirm the that the value `True` appears in the `READY` column when you run the following command:
++
+[source,terminal]
+----
+oc get smm default -n my-application
+----
++
+If you can access the `ServiceMeshMemberRoll` resource, you can also confirm that the `my-application` namespace is displayed in the `status.members` and `status.configuredMembers` fields of the `ServiceMeshMemberRoll` resource.
```

Comment:
Is this an alternative approach to achieve the same result, or is this a separate optional step with a slightly different result?

Reply:
Done